controller/setting_controller.go

```diff
@@ -218,7 +218,28 @@ func (sc *SettingController) syncSetting(key string) (err error) {
 	if err != nil {
 		return err
 	}
-	switch name {
+
+	if err := sc.syncNonDangerZoneSettingsForManagedComponents(name); err != nil {
+		return err
+	}
+	// SettingCategoryDangerZone and they will try to delete pods to update the settings.
```

Comment:
Remove this comment or make it understandable, or you can make a comment to `syncDangerZoneSettingsForManagedComponents`.

Reply:
Removed the comment. Thanks.